sqlglot/parser.py

```diff
@@ -1205,7 +1205,8 @@ def extend_props(temp_props: t.Optional[exp.Properties]) -> None:
             extend_props(self._parse_properties())
 
             self._match(TokenType.ALIAS)
-            begin = self._match(TokenType.BEGIN)
+            begin = self._match_text_seq("BEGIN")
+            #            begin = self._match(TokenType.BEGIN)
```

Comment:
why did you change this

Reply:
TokenType.BEGIN refers to Begin transaction where in `CREATE PROCEDURE` here in the create method we want BEGIN / END as in control flow ?